package.json

```diff
@@ -14,7 +14,8 @@
     "lint:md": "markdownlint ./*.md {content,src}/**/*.{md,mdx}",
     "prepare": "git submodule update --depth 1 --init && yarn build",
     "start": "gatsby develop --open",
-    "test": "prettier-package-json --list-different && yarn format:all --check && yarn lint && yarn tsc"
+    "test": "prettier-package-json --list-different && yarn format:all --check && yarn lint && yarn tsc",
+    "upgrade": "node utils/upgrade.js --components --containers --submodule"
```

Comment:
For consistency with other Garden JS scripts, please make `utils/upgrade.js` executable and remove `node` here.